.github/workflows/nx-cloud-agents.yml

```diff
@@ -51,16 +51,16 @@ jobs:
       # Set node/npm/yarn versions using volta, with optional overrides provided by the consumer
       - uses: volta-cli/action@fdf4cf319494429a105efaa71d0e5ec67f338c6e
         with:
-          node-version: '${{ inputs.node-version }}'
-          npm-version: '${{ inputs.npm-version }}'
-          yarn-version: '${{ inputs.yarn-version }}'
+          node-version: "${{ inputs.node-version }}"
```

Comment:
The default yml formatter in VSCode uses single quotes for these, so I think its preferable to stick with the automated tooling regardless of personal preference

Reply:
My VSCode didn't do any formatting. I'll switch to single quotes.